tuning/sft_trainer.py

```diff
@@ -100,6 +51,9 @@ def train(
         Union[peft_config.LoraConfig, peft_config.PromptTuningConfig]
     ] = None,
     trainer_controller_args: configs.TrainerControllerArguments = None,
+    tracker_config_args: Optional[Dict] = None,
```

Comment:
if default tracker is filelogger inside train() , dont you need fileloggingConfig default config here? why not set default value in training_args.tracker=FilElOggingTracker and default value of `tracker_config_args` to FileLoggingConfig.

inside code we need not set it then . If users have not overriden default arguments, they will get fileLogging by default. If they have overriden, we will use what user sets ? if they have explictyly overriden with None, then we can let it be None?

Reply:
@Ssukriti sure ... I made the Filelogging tracker explitct and passed from user point of view...default in the training args and tracker config default is also FileloggerTrackerConfig.